spec/03-plugins/syslog/log_spec.lua

```diff
@@ -0,0 +1,130 @@
+local utils = require "kong.tools.utils"
+local cjson = require "cjson"
+local helpers = require "spec.helpers"
+local exec = require("pl.utils").executeex
+
+describe("Syslog #ci", function()
+
+  local client, platform
+  
+  setup(function()
+    helpers.dao:truncate_tables()
+    assert(helpers.prepare_prefix())
+
+    local api1 = assert(helpers.dao.apis:insert {
+        request_host = "logging.com", 
+        upstream_url = "http://mockbin.com"
+      })
+    local api2 = assert(helpers.dao.apis:insert {
+        request_host = "logging2.com", 
+        upstream_url = "http://mockbin.com"
+      })
+    local api3 = assert(helpers.dao.apis:insert {
+        request_host = "logging3.com", 
+        upstream_url = "http://mockbin.com"
+      })
+  
+    assert(helpers.dao.plugins:insert {
+      api_id = api1.id,
+      name = "syslog", 
+      config = { 
+        log_level = "info", 
+        successful_severity = "warning",
+        client_errors_severity = "warning",
+        server_errors_severity = "warning"
+      },
+    })
+
+    assert(helpers.dao.plugins:insert {
+      api_id = api2.id,
+      name = "syslog", 
+      config = { 
+        log_level = "err", 
+        successful_severity = "warning",
+        client_errors_severity = "warning",
+        server_errors_severity = "warning"
+      },
+    })
+      
+    assert(helpers.dao.plugins:insert {
+      api_id = api3.id,
+      name = "syslog", 
+      config = { 
+        log_level = "warning", 
+        successful_severity = "warning",
+        client_errors_severity = "warning",
+        server_errors_severity = "warning"
+      },
+    })
+    
+    local success, code, _
+    success, code, platform, _ = exec("/bin/uname")
```

Comment:
I have a habit of occasionally doing this, for functions I don't use a lot. As a reminder that there is an extra return value (similar to dropping the first return value for example).

thx for flagging it.